rclcpp/test/rclcpp/test_init.cpp

```diff
@@ -1,66 +0,0 @@
-// Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
I didn't really dig into this, but do we have an equivalent test for this?

Reply:
I assume so, our coverage should have taken care of that. `rclcpp::ok()` is the alternative.

Reply:
`rclcpp::ok()` is covered in this file:

https://github.com/ros2/rclcpp/blob/cba6f20988758128377751de85dbc8351aeebefc/rclcpp/test/rclcpp/test_utilities.cpp